LEAKS.md

```diff
@@ -0,0 +1,17 @@
+# Memory leaks
```

Comment:
This is good info, but I don't think it belongs in the project. Please remove before we merge.